src/lib/y2users/linux/writer.rb

```diff
@@ -107,6 +116,17 @@ def write
       # @return [Y2User::Config]
       attr_reader :config
 
+      # Initial state of the system (usually a Y2User::Config.system in a running system) that will
+      # be compared with {#config} to know what changes need to be performed.
+      #
+      # @return [Y2User::Config]
+      attr_reader :initial_config
+
+      # The list of issues generated while writting changes to the system
+      #
+      # @return [Y2Issues::List]
+      attr_accessor :issues
```

Comment:
is accessor needed when you return it in write? it makes method hardly re-entrant. If there is reason, it is fine for me.

Reply:
Nope, there is no a reason apart from a _leftover_ from my side :)